app/models/tasks/evidence_submission_window_task.rb

```diff
@@ -68,4 +75,8 @@ def cancelled_schedule_hearing_task
       status: Constants.TASK_STATUSES.cancelled
     )
   end
+
+  def update_params_will_create_ihp_task?(params)
+    type == EvidenceSubmissionWindowTask.name && params[:status].eql?(Constants.TASK_STATUSES.completed)
```

Comment:
Is it possible for `type == EvidenceSubmissionWindowTask.name` to be `false` in our current codebase?

Reply:
I didn't catch this my first time through, but +1 to Yoom's question. I don't think this method can be called on anything that isn't of type EvidenceSubmissionWindowTask. I think we can remove this first check. (At a minimum, ESWTSpec doesn't fail if I remove it.)

```suggestion
    params[:status].eql?(Constants.TASK_STATUSES.completed)
```

Reply:
It can, when I throw a `byebug` in this method, `EvidenceSubmissionWindowTask.name` returns `true`, but `InformalHearingPresentationTask.name` for instance will return `false`

Reply:
I see what you're saying now however, the appeal seems to update to the correct state and the tests pass without the check. change accepted!